apis/v1alpha2/shared_types.go

```diff
@@ -166,9 +166,13 @@ type BackendRef struct {
 type RouteConditionType string
 
 const (
-	// This condition indicates whether the route has been admitted
-	// or rejected by a Gateway, and why.
+	// This condition indicates whether the route has been admitted or rejected
+	// by a Gateway, and why.
```

Comment:
MUST be set only if all rules are satisfied.

Reply:
Do you mean if all route rules are valid?

Reply:
And I think we may say in other places that a Route should only be not admitted if all routes are *not* valid. Regardless, this should tell a consistent story with the rest of the API, and should say it clearly.

Reply:
> Do you mean if all route rules are valid?

Yes. Is that not what we already want?

Reply:
So to clarify, what we want to say here is that admitted means that at least one Route rule has been admitted, not necessarily that the entirety of the Route has been admitted?

Reply:
Added some guidance around this as well.